### DISCUSSION TITLE
Cannot download Nred Kext

### DISCUSSION THREAD

**Discussion by Veir24** [ChefKissInc/NootedRed#191] [Q&A] [answered]
Okay guys, I am kind of a dummy. i wanted to install Mac OS on my acer nitro 5, so i wanted to download Nred Kext. I am new to github, so i dont see any option for downloading workflow archives in Actions section. A little help please

**Comment by jaywong966** [comment]
You can found it in Action tab

**Comment by VisualEhrmanntraut** [comment]
For future readers: since 22nd of May 2024 the nightly builds are now located as a "nightly" release instead of artefacts attached to the workflow run.

**Comment by VisualEhrmanntraut** [comment, marked as answer]
The nightly release was removed due to giving false impression of being notified of updates. The downloads are at GitHub Actions runs, and a login-free URL is provided via nightly.link in our website: https://chefkissinc.github.io.

https://chefkissinc.github.io/applehax/nootedred/
